microprofile/config-smallrye/src/test/java/org/wildfly/extension/microprofile/config/smallrye/Subsystem_1_0_ParsingTestCase.java

```diff
@@ -23,30 +23,16 @@
 package org.wildfly.extension.microprofile.config.smallrye;
 
 import static org.jboss.as.weld.Capabilities.WELD_CAPABILITY_NAME;
-import static org.junit.Assert.assertTrue;
-import static org.wildfly.extension.microprofile.config.smallrye.MicroProfileConfigExtension.CONFIG_SOURCE_PATH;
-import static org.wildfly.extension.microprofile.config.smallrye.MicroProfileConfigExtension.SUBSYSTEM_PATH;
-import static org.wildfly.extension.microprofile.config.smallrye.MicroProfileConfigExtension.VERSION_1_0_0;
 
 import java.io.IOException;
-import java.util.List;
 import java.util.Properties;
 
-import org.jboss.as.controller.ModelVersion;
-import org.jboss.as.controller.PathAddress;
-import org.jboss.as.model.test.FailedOperationTransformationConfig;
-import org.jboss.as.model.test.ModelTestControllerVersion;
-import org.jboss.as.model.test.ModelTestUtils;
 import org.jboss.as.subsystem.test.AbstractSubsystemBaseTest;
 import org.jboss.as.subsystem.test.AdditionalInitialization;
-import org.jboss.as.subsystem.test.KernelServices;
-import org.jboss.as.subsystem.test.KernelServicesBuilder;
-import org.jboss.dmr.ModelNode;
-import org.junit.Test;
 
-public class Subsystem_1_0_ParsingTestCase extends AbstractSubsystemBaseTest {
+public class MPConfigSubsystem_1_0_ParsingTestCase extends AbstractSubsystemBaseTest {
```

Comment:
Code does not compile, the file must also be renamed to `MPConfigSubsystem_1_0_ParsingTestCase.java`